optimize/backend/src/main/java/io/camunda/optimize/service/security/SessionService.java

```diff
@@ -99,51 +80,6 @@ private boolean isValidAuthToken(final String token) {
     return isValid;
   }
 
-  public Optional<String> refreshAuthToken(final String currentToken) {
```

Comment:
❓ So jwt is still used, but we dont do invalidation or refresh anymore? May be a bit out of touch with the jwt logic here can you give me a refresher

Reply:
In CCSM, we use the `CCSMAuthenticationCookieFilter` to handle the token refreshing. It is quite well commented in the flow.

The code being removed here was just the stuff that became unused after the deletion of the `AuthenticationCookieRefreshFilter`, which was only defined/used in the now-deleted platform web security adaptor